service/controller/clusterapi/v29/key/common.go

```diff
@@ -6,6 +6,23 @@ import (
 	"github.com/giantswarm/aws-operator/pkg/label"
 )
 
+func AWSTags(getter LabelsGetter, installationName string) map[string]string {
```

Comment:
I asked in Slack about the installation name weirdness. See https://gigantic.slack.com/archives/C2MS2VB37/p1562250052023800. Would maybe be cool to have this in our new `project` package.

Reply:
Should be relatively straight forward to get this information into the project package from the flags IMO.